Tutorials/whatiswot/11-Applications in Smart Homes and Smart Cities/README.md

```diff
@@ -0,0 +1,57 @@
+# Applications in Smart Homes and Smart Cities
+
+## Table of Contents
+
+- Smart Homes
+- Architecture Patterns
+  - Controllers
+  - Gateways
+- Smart Cities
+
+## Preliminary Script
+
+- In this two-part tutorial, we will introduce the W3C Web of Things by exploring its application domains and common architecture patterns. In this video our focus will be Smart Homes and Smart Cities.
+
+### Smart Homes
+
+- In smart homes, the ability to connect and manage multiple assets remotely over the network is highly beneficial.
+
+- The interesting part about smart homes isn’t tied to a specific solution you buy, but rather in how WoT enables flexibility. Manufacturers can provide Thing Descriptions (TDs) for their devices, which lets anyone build and customize their own systems. By following these open standards, devices can communicate with each other across brands, allowing for seamless integration. This approach ensures that devices remain adaptable and private, and don’t become obsolete, as they can continuously support new use cases and technologies.
+
+- This helps within various activities within the home, such as controlling lights, adjusting room temperature, managing home appliances, and enhancing security using smartphones. These capabilities enable optimization of energy and resource consumption.
+
+**Graphic:** `House with many WoT-powered devices, showcasing Thing Description of the house`
+
+#### Architecture patterns
+
+In smart homes, we can already notice common architecture patterns. In this video, we will explain Controller and Gateway patterns.
+
+1. **Controllers:**
+
+- A common pattern involves using a controller, both locally and remotely, to interact with devices via the home network by consuming the Thing Description.
```

Comment:
```suggestion
- A common pattern involves using a controller, locally and/or remotely, to interact with devices on the home network by consuming the Thing Description.
```

Reply:
We probably need to adjust the voice generator's text.